pages/Profile/index.tsx

```diff
@@ -1,13 +1,49 @@
-import React from "react";
+import { style } from '@mui/system';
+import React from 'react';
 import Layout from "../../components/Layout/Layout";
 import styles from "./Profile.module.css";
 
 const ProfileSettingsPage: React.FunctionComponent = () => {
-  return (
-    <Layout title="Profile">
-      <main className={styles.main}>Profile Settings</main>
-    </Layout>
-  );
-};
+    
+    return (
+        <body className = {styles.body}>
+            <Layout title = 'Profile'>
+                <div className = {styles.all}>
+                    <div className = {styles.header}>
+                        <h2>Profile Settings</h2>
+                    <div className = {styles.namebar}>
+                        <h2>Cindy Zhang</h2>
+                        <button className={styles.button}>Edit Profile</button>
+                    </div>
+                    <hr></hr>
+                    <div className = {styles.boxes}>
+                        <div className = {styles.box}>
```

Comment:
You should the code within the `div` styled with `styles.box` its own component within the `components` folder. We want to make front-end components that are reusable/modularized. Another possibility, is also just using the Material UI [Card](https://mui.com/components/cards/) component. Let me know if you need help with any of this or if you have any questions!